tests/docker-scripts/Dockerfile

```diff
@@ -50,4 +42,4 @@ RUN SKIP_POSTINSTALL=1 npm ci
 COPY . .
 
 # Now build the app
-RUN npm run build
\ No newline at end of file
+RUN VITE_PLAYWRIGHT=1 npm run build
```

Comment:
Could change this env variable to something more descriptive like `DISABLE_FONT_FEATURES` or something. I don't think the `VITE_` is necessary since we load all env variables in our `vite.config.ts`, but doesn't hurt to have it.

Reply:
> To prevent accidentally leaking env variables to the client, only variables prefixed with VITE_ are exposed to your Vite-processed code. e.g. for the following env variables

> https://vitejs.dev/guide/env-and-mode.html#env-files

I assumed it was needed from this. I could name it `PLAYWRIGHT_CSS` if you think that's better? I was leaving it slightly more generic incase we need to slip in any other css to mask anything else in playwright tests ever.

Reply:
It normally is (and their recommendation makes sense), but we change that in the vite config and everything is available. We only use it for `npm_package_version` though I think.

`VITE_PLAYWRIGHT_CSS` works